task-1/src/Triangle.ts

```diff
@@ -0,0 +1,40 @@
+import { Shape } from './Shape';
+import { Point } from './Point';
+
+export class Triangle extends Shape {
+    constructor(point1: Point, point2: Point, point3: Point, color?: string, filled?: boolean) {
+        super([point1, point2, point3], color, filled);
+        this.points = [point1, point2, point3];
+    }
+
+    private getPointsValues(): string {
+        return this.points.reduce((result: string, point: Point, index: number) => {
+            return result += `v${index + 1}=${point.toString()}` + `${index === 2 ? '' : ','}`
+        }, '');
+    }
+
+    toString(): string {
+        return `Triangle[${this.getPointsValues()}]`;
+    }
+
+    private roundedDistance(point: Point, index: number) {
```

Comment:
As was mentioned above, it can be replaced with `toPrecision` inside of the distance